corpus/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,63 @@
+{% extends 'base.html' %}
+
+{% block title %}
+    Reset Password
+{% endblock %}
+
+{% block content %}
+    <div class="card w-3/4 lg:w-1/3 mx-auto my-10 bg-base-200 shadow-xl">
+        <div class="card-body">
+            {% if validlink %}
+                <h1 class="card-title text-2xl">Reset Password</h1>
+                <p>
+                    Resetting Password for {{ user }}
+                </p>
+                <form method="post">
+                    {% csrf_token %}
+
+                    {% if form.non_field_errors %}
+                        {% for error in form.non_field_errors %}
+                            <div role="alert" class="alert alert-error mt-1">
+                                <span>{{ error }}</span>
+                            </div>
+                        {% endfor %}
+                    {% endif %}
+
+                    <div class="w-full my-2">
+                        <label for="{{ form.new_password1.id_for_label }}">New Password</label>
+                        {{ form.new_password1 }}
+                        {% if form.new_password1.errors %}
+                            <div class="my-1">
+                                <div role="alert" class="alert alert-error">
+                                    {{ form.new_password1.errors }}
+                                </div>
+                            </div>
+                        {% endif %}
+                    </div>
+
+                    <div class="w-full my-2">
+                        <label for="{{ form.new_password2.id_for_label }}">Confirm New Password</label>
+                        {{ form.new_password2 }}
+                        {% if form.new_password2.errors %}
+                            <div class="my-1">
+                                <div role="alert" class="alert alert-error">
+                                    {{ form.new_password2.errors }}
+                                </div>
+                            </div>
+                        {% endif %}
+                    </div>
+
+                    <div class="card-actions mt-5">
+                        <button type="submit" class="btn btn-primary btn-block">Send me a reset link</button>
```

Comment:
Update the button text. This button will reset their password, not send them another reset link.